src/test/java/org/folio/rest/impl/PurchaseOrderLinesApiTest.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.rest.impl;
+
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+
+public class PurchaseOrderLinesApiTest extends TestBase {
+
+  private final Logger logger = LoggerFactory.getLogger(OrdersAPITest.class);
+  private static final String PO_LINES_ENDPOINT = "/orders-storage/po-lines";
+
+  private final String poLineSample = getFile("data/po-lines/312325-1_fully_received_electronic_method-purchase_at_vendor_system.json");
```

Comment:
Please use TestEntities.PO_LINE to get data file and endpoint

Reply:
Done